manual/cops_rspec.md

```diff
@@ -0,0 +1,3392 @@
+# RSpec
```

Comment:
Please remove this file from the PR

We recently switched from using the `manual/` folder to using the `docs/` folder, and this file was probably created somewhere in the middle. Sorry about the confusion 😄